config/routes.rb

```diff
@@ -7,13 +7,13 @@
   root 'home#index'
 
   # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
+  get 'discover' => 'discover#index'
 
   # Example of named route that can be invoked with purchase_url(id: product.id)
   #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
 
   # Example resource route (maps HTTP verbs to controller actions automatically):
-  #   resources :products
+  #   resources :discover
```

Comment:
Don't need to define discover as a resource... we just have one route which was defined in line 10.